meta-genivi-dev/meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0+20170516.bb

```diff
@@ -0,0 +1,22 @@
+# Recipe created by recipetool
+# This is the basis of a recipe and may need further editing in order to be fully functional.
```

Comment:
Since you edited the recipe after generation and know it to be fully functional already, this whole comment should IMO be removed.